extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/simulate/KafkaResource.java

```diff
@@ -19,102 +19,48 @@
 
 package org.apache.druid.indexing.kafka.simulate;
```

Comment:
I think we had missed renaming this package in the first PR. Let's rename this to `kafka.embedded` or something.

Reply:
Do you want to do it in this PR or in another, clean one that doesn't change anything else?

Reply:
We can do it later too 👍🏻